optaplanner-docs/src/main/asciidoc/Chapter-Partitioned_search/Chapter-Partitioned_search.adoc

```diff
@@ -8,27 +8,214 @@
 :sourcedir: .
 
 
-[[partitionedSearchOverview]]
-== Overview
+[[partitionedSearchAlgorithm]]
+== Algorithm Description
 
-For very big datasets, it is sometimes worthwhile to partition the datasets into smaller pieces.
+For big datasets, usually above 5000 planning entities,
+it is usually worthwhile to partition the datasets into smaller pieces and solve those separately.
```

Comment:
Suggestion: "It is usually more efficient to partition large data sets (usually above 5000 planning entities) into smaller pieces and solve them separately."